apps/web/src/services/dashboards/widgets/asset-widgets/compliance-check-status/ComplianceCheckStatusWidget.vue

```diff
@@ -133,126 +139,98 @@ interface InnerChartData {
-        start: dayjs.utc(state.settings?.date_range?.start).format(DATE_FORMAT),
-        end: dayjs.utc(state.settings?.date_range?.end).format(DATE_FORMAT),
+        start: dayjs.utc(state.settings?.date_range?.start).subtract(1, 'month').format(DATE_FORMAT),
+        end: dayjs.utc(state.settings?.date_range?.start).format(DATE_FORMAT),
     })),
-    complianceData: computed<ComplianceData>(() => state.data?.compliance ?? {}),
-    severityData: computed<SeverityData[]>(() => state.data?.severity ?? []),
     outerChartData: computed<OuterChartData[]>(() => COMPLIANCE_STATUS_MAP_VALUES.map((status) => ({
         status: status.label,
-        value: state.checkCount[status.name],
-        pieSettings: {
-            fill: color(status.color),
-            stroke: color(status.color),
-        },
-    }))),
-    innerChartData: computed<InnerChartData[]>(() => SEVERITY_FAIL_STATUS_MAP_VALUES.map((status) => ({
-        severity: status.label,
-        value: state.severityData.find((data) => data.severity === status.name)?.fail_finding_count ?? 0,
+        value: state.checkCount?.[status.name],
         pieSettings: {
             fill: color(status.color),
             stroke: color(status.color),
         },
     }))),
+    innerChartData: computed<InnerChartData[]>(() => {
+        const targetDataList = state.data?.filter((d) => d.date === state.dateRange.end && d.key === 'fail_finding_count') ?? [];
+        return targetDataList.map((data) => ({
+            severity: SEVERITY_STATUS_MAP[data.severity].label,
+            value: data.value,
+            pieSettings: {
+                fill: color(SEVERITY_STATUS_MAP[data.severity].color),
+                stroke: color(SEVERITY_STATUS_MAP[data.severity].color),
+            },
+        }));
+    }),
     //
-    accountCount: computed(() => state.complianceData?.account_count ?? 0),
-    complianceCount: computed(() => state.complianceData?.compliance_count ?? 0),
-    checkCount: computed(() => ({
-        [COMPLIANCE_STATUS_MAP.PASS.name]: state.complianceData?.pass_check_count ?? 0,
-        [COMPLIANCE_STATUS_MAP.FAIL.name]: state.complianceData?.fail_check_count ?? 0,
-    })),
-    score: computed(() => Math.round(state.complianceData?.score ?? 0)),
+    accountCount: 0, // TODO: should be changed to real data when api is ready
+    prevComplianceCount: computed<number|undefined>(() => {
+        if (!state.data) return undefined;
+        const targetDataList = state.data.filter((d) => d.date === state.dateRange.start && d.key === 'compliance_count');
+        return sum(targetDataList.map((d) => d.value));
+    }),
+    complianceCount: computed<number|undefined>(() => {
+        if (!state.data) return undefined;
+        const targetDataList = state.data.filter((d) => d.date === state.dateRange.end && d.key === 'compliance_count');
+        return sum(targetDataList.map((d) => d.value));
+    }),
+    complianceCountHelpText: computed<TranslateResult|undefined>(() => {
```

Comment:
suggestion: Please change the state name more meaningful~ such as complianceCountComparingMessage